synapse/lib/snap.py

```diff
@@ -805,9 +789,8 @@ async def _joinSodes(self, buid, sodes):
         })
 
         node = s_node.Node(self, pode, bylayer=bylayer)
-        if self.cachebuids:
```

Comment:
The `cachebuids` attribute is used in the StormDmon to disable node caching for long lived snaps. We'll need to discuss another way to allow nodes to expire from the cache if are removing the option to control that behavior.

Reply:
Good catch, thanks! I'll have a look :+1: